api/data_refinery_api/serializers.py

```diff
@@ -250,7 +250,6 @@ class Meta:
             "sha1",
             "s3_bucket",
             "s3_key",
-            "s3_url",
```

Comment:
This can stay! Some computed files can be downloaded with just an S3_url such as transcriptome indices and quantile normalization targets. They are in their own buckets with different permissions.